aether/sdk/drf/views.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (C) 2019 by eHealth Africa : http://www.eHealthAfrica.org
+#
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from django.db import transaction
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from rest_framework import status
+from django.utils.translation import ugettext as _
+
+
+class FilteredMixin(object):
+
+    @transaction.atomic
+    @action(detail=False, methods=['delete'], url_path='filtered-delete')
+    def filtered_delete(self, request, *args, **kwargs):
+        '''
+        This adds a bulk delete action based on the supplied filters to a viewset
+        accessible via:
+
+        `/{modelname}/filtered-delete/?{query_string}`
+
+        All delete operations are rolledback if any error is encountered
```

Comment:
"rolledback" or "rolled back"?

Reply:
:arrow_double_up:

Reply:
been fixed

Reply:
You fixed that in the update method, not in the delete method... :wink: